src/client/aureliaCLICommands.ts

```diff
@@ -12,6 +12,7 @@ export default class AureliaCliCommands {
       vscode.commands.registerCommand('extension.auGenerate', () => this.auGenerate(outputChannel)),
       vscode.commands.registerCommand('extension.auTest', () => this.runCommand(['test'], outputChannel, false)),
       vscode.commands.registerCommand('extension.auBuild', () => this.auBuild(outputChannel)));
```

Comment:
Thanks for the pull request!

If you add the line above the auBuild line or change the above line to end on ```(outputChannel)),``` and the new line on ```outputChannel, true)));``` then the build will succeed.

You could test if the build succeeds by running ```npm run build``` on your local machine, once you done a ```npm install```.

Then we can take a look and test it :-)